generators/entity-server/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -87,7 +87,7 @@
  */
 @RunWith(SpringRunner.class)
 <%_ if (authenticationType === 'uaa' && applicationType !== 'uaa') { _%>
-@SpringBootTest(classes = {<%= mainClass %>.class, SecurityBeanOverrideConfiguration.class})
+@SpringBootTest(classes = {SecurityBeanOverrideConfiguration.class, <%= mainClass %>.class})
```

Comment:
Configuration files are now order sensitive. bug in latest Spring Boot libs?

Reply:
Never thought of this, great work!